### ISSUE TITLE
Pagination in Azure using node

### ISSUE THREAD

**Issue by ghost** [Azure/azure-sdk-for-node#542]
I am writing an application in Azure and node js .This will take read all the data from the Azure and writes in a array. I will write all the data that the query has provided in  a Array.

so, Based on my condition I have, I will display to the user say example 11 to 20.

The table can have more than 50 elements.

so when the user submits Next, I want to send a query with

pageContinuation.nextPartitionKey and pageContinuation.nextRowKey directly from the DB.

As my initial query is from where I am reading all the entries is with out the pageContinuation I will not have the data which is needed to query the next values.

http://msdn.microsoft.com/en-us/library/windowsazure/dd135718.aspx

I have seen that : "hash of the next partition key to be returned"

In this case how can get the values, how to do a  hash and get the values? what Hash algorithm does this mean.

In conclusion,

I have query response with all partition keys and Row Keys.

After that I want to construct a new Query to get a particular elements from Azure, for that I need to know  pageContinuation.nextRowKey and  pageContinuation.nextPartitionKey how will get them. I cannot get these values in my initial Query because I dont know where I am ending the data so I need to read all data.

I am using Nodejs and Azure on Ubantu.
